packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -182,7 +182,7 @@ export class Button extends FormAssociatedButton {
     /**
      * References the root element
      */
-    public root: HTMLButtonElement;
+    public control: HTMLButtonElement;
```

Comment:
this is *technically* a breaking change actually :/ We probably need to do this in an additive way or not make the change to align to "control"

Reply:
@nicholasrice @chrisdholt Gotcha. What would be the best path forward?

Reply:
I mean - this was ultimately published end of last week. Would it be reasonable to see if folks have even used this? Otherwise we take an issue IMO and fix later.

Reply:
I know *I* have used it, but perhaps we should ask others.

Reply:
I think my preference is to stick to the policy on this one and not break the API surface. Something like this can be done to get the best of both worlds:

```ts
 /**
  * @deprecated
  */
  public get root() { return this.control; }
  
 /**
 * References the root element
 */
  public control: HTMLButtonElement;
```

we could maybe add a note that `control` should be `@internal` as well - while it's documented now, it really *shouldn't* be because it leaks implementation details to the public API.

Reply:
Thanks @nicholasrice!

Reply:
@nicholasrice looks like this was updated - can you take another pass when you have a minute?